src/components/main/note/final-view/NoteFinalView.js

```diff
@@ -80,37 +79,66 @@ const NoteFinalView = ({ note, id, userId, newPageX, newPageY }) => {
       },
       _id: id,
```

Comment:
I think it's some old data you got from the old graasp. If you can, remove all these `_id` to only use `id`